classes/module/Module.php

```diff
@@ -1275,19 +1275,11 @@ public static function getModulesOnDisk($use_config = false, $logged_on_addons =
                         $file = substr($file, 0, -2);
```

Comment:
In order to recognize php content, we need to keep the `<?php` and `?>` tags. Please remove the lines 1270 to 1276.